canonical-web-and-design/ubuntu.com
# Build CUBE landing page

## Done

- Build CUBE landing page at `/cube`


## QA

- Visit the demo at: https://ubuntu-com-8925.demos.haus/cube
- Check the page against this [design](https://app.zeplin.io/project/5fc7a466de658e0cb99276c8/screen/5fd355ed883fc9481de84a69)


## Issue / Card

Fixes [#3498](https://github.com/canonical-web-and-design/web-squad/issues/3498)

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/102204214-7b1c6200-3ec1-11eb-8edc-29f0ccd5acb4.png)
